zilencer/models.py

```diff
@@ -36,6 +36,10 @@ class RemoteZulipServer(models.Model):
     contact_email: str = models.EmailField(blank=True, null=False)
     last_updated: datetime.datetime = models.DateTimeField("last updated", auto_now=True)
 
+    # For deactivating a self-hosted installation's registration for
+    # our Mobile Push Notifications Service
```

Comment:
I think this can be briefer -- the whole record is for that service.  "Whether the server registration has been deactivated."